base/logger/R/logifnot.R

```diff
@@ -74,6 +79,7 @@ debugifnot <- function(msg, ...) {
 }
 
 #' Check a list of conditions
+#’ @param ... other arguments passed on to check_conditions
```

Comment:
Here's the much simpler fix for the issue I was trying to fix with all the extra `@param ...` lines: This and `is_definitely_true` below are both very simple helper functions that aren't exported and aren't used anywhere outside this file, so they don't need Roxygen documentation at all! Let's just convert these from Roxygen comments (`#'`) to plain old R comments (`#`).

After making this change, you'll need to re-run Roxygen so make it delete the Rd files for these -- easiest will be to use `/document` and let the GitHub robot do it for you.

```suggestion
# Check a list of conditions
```

Reply:
@infotroph understood, but what if these functions **were** used by any outside functions. What changes could fix this warning?

Reply:
Then we'd either need to add parameter descriptions here, or tell Roxygen to use the descriptions from somewhere else using one of [`@inheritParams`](https://roxygen2.r-lib.org/articles/rd.html#inheriting-documentation-from-other-topics), [`@describeIn`](https://roxygen2.r-lib.org/articles/rd.html#describein), or [`@rdname`](https://roxygen2.r-lib.org/articles/rd.html#rdname).